src/rust/graph-descriptions/Cargo.toml

```diff
@@ -18,8 +18,17 @@ serde_json = "1"
 serde_derive = "1"
 
 derive_builder = "0.9"
+dgraph-tonic = "0.8"
 
 log = "0.4"
+dgraph_query_lib = { git = "https://github.com/d0nutptr/dgraph_query_lib", branch = "main" }
```

Comment:
Can we either vendor this into the monorepo, like in a `/vendored/` directory (ask in slack imo), or get this on crates.io? At minimum we'd want to specify a hash, but I think to get into staging even that would not really be possible.

Reply:
Yea I agree with that. I'll put it on crates.io eventually so that'll probably be the move